src/components/Wrappers/withMobileDetection.js

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'preact';
+import { html } from 'htm/preact';
+
+export default function withMobileDetection(WrappedComponent) {
+  return class extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        isMobile: false,
+        daysRange: 70,
```

Comment:
`daysRange: 70` is in lines 10 and 20, could we put it into a variable?

Reply:
Yeah, I forgot change that to use instead the props hahaha